internal/codegen/template.go

```diff
@@ -112,6 +124,12 @@ func (t *Template) Render(st *Stencil, vals *Values) error {
 		return err
 	}
 
+	// If we're a library template, we don't want to generate any files so
```

Comment:
note to self, fix function comment. Now that we gated `file` calls we could also probably omit this in general.